Update generate docs (#16825)
cloudflare/cloudflare-docs: src/content/docs/workers/wrangler/commands.mdx

- `NAME` string optional (default: name of working directory)
  - The name of the Workers project. This is both the directory name and `name` property in the generated `wrangler.toml` [configuration](/workers/wrangler/configuration/) file.
- `TEMPLATE` string optional

>   - The URL of a GitHub template, defaults to using `npm create cloudflare` if no template is provided.

Reviewer: ```suggestion
  - The URL of a GitHub template. If no template is provided, calling `wrangler generate` will instead call `npm create cloudflare`.
```
